getting_started/Data_Input_Pipeline.md

```diff
@@ -135,3 +134,8 @@ eval_split: 'validation'
 # TFDS input pipeline only supports tokenizer in spm format
 tokenizer_path: "assets/tokenizer.llama2"
 ```
+3. Enable determinism (optional)
+```
+tfds_iter_checkpointing: True
+```
+Note that deteriminism with preemption requires checkpointing the data iterator, and the checkpoints will be larger in size. 
```

Comment:
Could you add more details on what contributes to the size of the checkpoint? Would be good to provide a way to estimate the size to help the users make plans